intersphinx_registry/registry.json

```diff
@@ -14,6 +16,7 @@
   "cffi": ["https://cffi.readthedocs.io/en/latest/", null],
   "click": ["https://click.palletsprojects.com/", null],
   "conda": ["https://conda.io/en/latest/", null],
+  "ConfigSpace": ["https://automl.github.io/ConfigSpace/latest/", null],
```

Comment:
```suggestion
  "configspace": ["https://automl.github.io/ConfigSpace/latest/", null],
```

Reply:
Think this might be case sensitive.

Reply:
I guess I was thinking the mapping on the left hand side should match the python import statements.
But I didn't test it 😅

Reply:
I'm thorn, because `ipython` is lower case in the registry; it's the same with `_`, and `-`, which is also normalized by PyPI.

I think we can keep either; and do some normalizing; fallback if case is different.

It's tough, because there is not right or wrong answers.

Let's do as is for now; (thanks for the PR), and we'll update casing later if needs be with a deprecation periode.